.travis.yml

```diff
@@ -27,11 +27,26 @@ install:
 # command to run tests
 script:
   - ./testsuite/MDAnalysisTests/mda_nosetests -v --with-coverage --cover-package MDAnalysis --processes=2 --process-timeout=120 --with-memleak
-  - test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_BRANCH == "develop" && cd ./package && python setup.py build_ext --inplace && cd ./doc/sphinx && make clean html || true
+  - |
+     test ${TRAVIS_PULL_REQUEST} == "false" && \
+     test ${TRAVIS_BRANCH} == ${GH_DOC_BRANCH} && \
+     test "${TRAVIS_BUILD_NUMBER}.1" == "${TRAVIS_JOB_NUMBER}" && \
+     (cd ./package && python setup.py build_ext --inplace) && \
+     (cd ./package/doc/sphinx && make clean html) || \
+     true
 after_success:
   - coveralls
-  - test $TRAVIS_PULL_REQUEST == "false" && test $TRAVIS_BRANCH == "develop" && bash ./maintainer/deploy_docs.sh
+  - |
+     test ${TRAVIS_PULL_REQUEST} == "false" && \
+     test ${TRAVIS_BRANCH} == ${GH_DOC_BRANCH} && \
+     test "${TRAVIS_BUILD_NUMBER}.1" == "${TRAVIS_JOB_NUMBER}" && \
+     bash ./maintainer/deploy_docs.sh
   
 env:
    global:
-      secure: "HIj3p+p2PV8DBVg/KGUx6n83KwB0ASE5FwOn0SMB9zxnzAqe8sapwdBQdMdq0sXB7xT1spJqRxuxOMVEVn35BNLu7bxMLfa4287C8YXcomnvmv9xruxAsjsIewnNQ80vtPVbQddBPxa4jKbqgPby5QhhAP8KANAqYe44pIV70fY="
\ No newline at end of file
+      - secure: "HIj3p+p2PV8DBVg/KGUx6n83KwB0ASE5FwOn0SMB9zxnzAqe8sapwdBQdMdq0sXB7xT1spJqRxuxOMVEVn35BNLu7bxMLfa4287C8YXcomnvmv9xruxAsjsIewnNQ80vtPVbQddBPxa4jKbqgPby5QhhAP8KANAqYe44pIV70fY="
+      - GH_DOC_BRANCH: develop
+      - GH_REPOSITORY: github.com/MDAnalysis/mdanalysis.git
+      - GIT_CI_USER: TravisCI
```

Comment:
I would have liked to be able to say
```yaml
 - GIT_CI_USER: "Travis CI"
```
but neither backslash quoting nor quotes around retained the space. Travis reported always that GIT_CI_USER=Travis. In any case, it does not matter...